packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -974,9 +974,12 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   setUpLocusMediaSharesListener() {
+    // Will get triggered on local and remote share
     this.locusInfo.on(EVENTS.LOCUS_INFO_UPDATE_MEDIA_SHARES, (payload) => {
       const {contentId, disposition} = payload.current;
 
+      if (this.statsAnalyzer) this.statsAnalyzer.updateShareStatus(disposition === FLOOR_ACTION.GRANTED);
```

Comment:
Please space this out so eslint doesn't complain or use the optional chaining method:

```suggestion
      this.statsAnalyzer?.updateShareStatus(disposition === FLOOR_ACTION.GRANTED);
```

Reply:
@adamweeks i got a lint error when i do that , Keeping it like that

Reply:
I've noticed the eslint error as well when trying to use the optional chaining. We'll need to update the eslint config. In the meantime, you can either disable the eslint warning or move this to a multi line `if`:

```suggestion
      if (this.statsAnalyzer) {
        this.statsAnalyzer.updateShareStatus(disposition === FLOOR_ACTION.GRANTED);
      }
```